### ISSUE TITLE
[20.08] fails to build from source: error: 'sys_siglist' undeclared

### ISSUE THREAD

**Issue by tgurr** [greenbone/gvmd#1279]
### Expected behavior

build.

### Actual behavior

```
$ make
[  0%] Building C object src/CMakeFiles/gvmd.dir/main.c.o
[  5%] Building C object src/CMakeFiles/gvmd.dir/gvmd.c.o
In file included from /usr/x86_64-pc-linux-gnu/include/glib-2.0/glib.h:62,
                 from /home/user/Downloads/gvmd-gvmd-20.08/src/gvmd.c:75:
/home/user/Downloads/gvmd-gvmd-20.08/src/gvmd.c: In function 'serve_and_schedule':
/home/user/Downloads/gvmd-gvmd-20.08/src/gvmd.c:1337:20: error: 'sys_siglist' undeclared (first use in this function)
 1337 |                    sys_siglist[termination_signal]);
      |                    ^~~~~~~~~~~
/usr/x86_64-pc-linux-gnu/include/glib-2.0/glib/gmessages.h:346:32: note: in definition of macro 'g_debug'
  346 |                                __VA_ARGS__)
      |                                ^~~~~~~~~~~
/home/user/Downloads/gvmd-gvmd-20.08/src/gvmd.c:1337:20: note: each undeclared identifier is reported only once for each function it appears in
 1337 |                    sys_siglist[termination_signal]);
      |                    ^~~~~~~~~~~
/usr/x86_64-pc-linux-gnu/include/glib-2.0/glib/gmessages.h:346:32: note: in definition of macro 'g_debug'
  346 |                                __VA_ARGS__)
      |                                ^~~~~~~~~~~
make[2]: *** [src/CMakeFiles/gvmd.dir/build.make:95: src/CMakeFiles/gvmd.dir/gvmd.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:171: src/CMakeFiles/gvmd.dir/all] Error 2
make: *** [Makefile:182: all] Error 2
```

### GVM versions

**openvas-scanner:** 20.8.0

**gvm-libs:** 20.8.0

**GCC:** gcc (GCC) 10.2.0
**glibc:** 2.32
**glib:** 2.64.5

### Environment

**Operating system:** Linux

**Installation method / source:** (packages/source installation)

**Comment by tgurr** [comment]
From the release notes of glibc 2.32, https://sourceware.org/pipermail/libc-announce/2020/000029.html

* The deprecated arrays sys_siglist, _sys_siglist, and sys_sigabbrev
  are no longer available to newly linked binaries, and their declarations
  have been removed from <string.h>.  They are exported solely as
  compatibility symbols to support old binaries.  All programs should use
  strsignal instead.

**Comment by cfi-gb** [comment]
Fixed by @tgurr in #1280, thanks for this contribution :+1:
